troposphere/static/js/components/projects/detail/resources/InstanceBatchActionButtons.jsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import Backbone from "backbone";
+// NOTE raven-js included during evaluation
+// - may be removed after ATMO-981 resolved
```

Comment:
Raven was only intended to be in here during the evaluation within the testcloud.